lib/components/user/form-navigation-buttons.tsx

```diff
@@ -15,19 +15,24 @@ const RightButton = styled(Button)`
   float: right;
 `
 
-interface ButtonType {
+export interface ButtonType {
+  /** The component to use for the button. */
+  ButtonComponent?: ReactNode
```

Comment:
Can we use `children` instead?

Reply:
Or `content`?

Reply:
Good suggestions I think either make sense. I'll change to `content`

Reply:
No I meant using `children` so you don't have to pass a prop at all and can instead just use the child of the component!

Reply:
I understood how to implement this only halfway, but having trouble subsequently rendering children of extraButton in SavedTripEditor